tests/blockchain.rs

```diff
@@ -415,11 +417,11 @@ fn can_read_state_from_initial_storage_slots() {
         Opcode::LW(0x10, 0x10, 0),
         // alloc 32 bytes stack space
         Opcode::ADDI(0x11, REG_SP, 0),
-        Opcode::CFEI(32 as Immediate24),
+        Opcode::CFEI(32),
```

Comment:
Mixed feelings in this one as well. I wanted to make it clear there that we are using immediate values - and not register references.